### PR TITLE
Default content profile for stories. [SDCP-100]

### PR DESCRIPTION
**Pull request by GyanP**
SDCP-100 and SDCP-104

### CODE REVIEW

File: server/settings.py [hunk 1]
```diff
@@ -67,5 +67,10 @@ def env(variable, fallback_value=None):
 HIGHCHARTS_SERVER_HOST = env('HIGHCHARTS_SERVER_HOST', 'localhost')
 HIGHCHARTS_SERVER_PORT = env('HIGHCHARTS_SERVER_PORT', '6060')
 
+LANGUAGES = [
+    {'language': 'en-CA', 'label': 'English', 'source': True, 'destination': True},
```
Comment (petrjasek) [inline server/settings.py]: shouldn't it be `en_CA` and `fr_CA` ?

Comment (GyanP) [reply server/settings.py]: I am not sure, it was as such in the ticket https://dev.sourcefabric.org/browse/SDCP-104

Comment (GyanP) [reply server/settings.py]: and it's similar in ntb also https://github.com/superdesk/superdesk-ntb/blob/master/server/settings.py#L183

Comment (karelpetrak) [reply server/settings.py]: Hi @petrjasek afaik we stick to hyphens, it's also used in newsml input/output, e.g. <language tag="en-US" />, and by Tansa

**Pull request merged**